Source/uis/theme/draw.lua

```diff
@@ -0,0 +1,117 @@
+-- theme/draw
+
+return function()
+	font_ui:print(L.THEMESTITLE, 8, 8+4)
+
+	font_ui:print(L.ACTIVETHEMES, 8, 8+4+24)
+
+	local should_disable = {}
+	local should_enable = {}
+	local should_shift_up = {}
+	local should_shift_down = {}
+
+	local x = 8
+	local y = 8 + 4 + 24 + 32
+	local active_themes = theme:get_active_themes()
+	local inactive_themes = {}
+	for name, theme in pairs(theme:get_themes()) do
+		if not table.contains(active_themes, name) then
+			table.insert(inactive_themes, name)
+		end
+	end
+
+	for i = #active_themes, 1, -1 do
+		local name = active_themes[i]
+		love.graphics.setColor(255, 255, 255)
+
+		if i ~= 1 then
+			hoverrectangle(128,128,128,128, x, y, 16, 16)
+			font_ui:printf("X", x, y + 4, 16, "center")
+
+			if (not mousepressed) and nodialog and mouseon(x, y, 16, 16) and love.mouse.isDown("l") then
+				mousepressed = true
+				table.insert(should_disable, name)
+			end
+
+			if i ~= #active_themes then
+				hoverrectangle(128,128,128,128, x + 20, y, 16, 16)
+				font_ui:printf(arrow_up, x + 20, y + 4, 16, "center")
+				if (not mousepressed) and nodialog and mouseon(x + 20, y, 16, 16) and love.mouse.isDown("l") then
+					mousepressed = true
+					table.insert(should_shift_up, name)
+				end
+			end
+
+			if i ~= 2 then
+				hoverrectangle(128,128,128,128, x + 40, y, 16, 16)
+				font_ui:printf(arrow_down, x + 40, y+4, 16, "center")
+				if (not mousepressed) and nodialog and mouseon(x + 40, y, 16, 16) and love.mouse.isDown("l") then
+					mousepressed = true
+					table.insert(should_shift_down, name)
+				end
+			end
+		end
+
+		local theme_info = theme:get_themes()[name].info
+
+		font_ui:print(theme_info.name .. " by " .. theme_info.author, x + 48 + 16, y + 4)
+		y = y + 24
+	end
+
+	love.graphics.setColor(128, 128, 128)
+
+	y = y + 24
+
+	love.graphics.setLineWidth(2)
+	love.graphics.setLineStyle("rough")
```

Comment:
The line style is not reset to "smooth" again (I have no idea how different this looks in other places in the code but drawing state should not change by going to a certain screen)